.gitignore

```diff
@@ -65,3 +65,6 @@ typings/
 
 # JetBrains based IDEs, such as WebStorm
 .idea
+
+# Visual Studio Code
+.history/
```

Comment:
**suggestion** if you are using vscode you should put this to your [global gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#create-a-global-gitignore)

Reply:
This is exactly what I was needing, thank you so much!
I just recently read on a projectsolid PR that one should actually NOT add to the repo gitignore such stuff... but to the local one. Unfortunately, I added it already to other repos too 😓
I will make the according changes.